functions/src/modules/SE.functions.ts

```diff
@@ -34,14 +32,13 @@ export const uploadProcessing = functions.storage.object()
 
```

Comment:
Use named capture groups to get the list and the task id here.

Reply:
It's not supported yet, we need to use a package .. I tried using this [package](https://www.npmjs.com/package/named-regexp-groups) but it seems to have some issues.
Do you have a particular suggestion .. or should I move on testing more packages ?